src/components/chip/themes/chip.indigo.scss

```diff
@@ -0,0 +1,140 @@
+@use './chip.base.scss';
+@use '../../../styles/utilities' as *;
+
+:host button {
+    border: 1px solid color(gray, 400);
+    background: color(surface, 500);
+    color: color(gray, 700);
+
+    &:hover {
+        background: color(gray, 200);
+    }
+
+    &:focus {
+        outline: 3px solid color(gray, 300);
+    }
+
+    &[disabled] {
+        border-color: color(gray, 200);
+        color: color(gray, 500);
+    }
+}
+
+:host([selected]) button:not([disabled]) {
+    border-color: transparent;
+    background: color(primary, 500);
+    color: contrast-color(primary, 700);
+
+    &:hover {
+        background: color(primary, 600);
+    }
+
+    &:focus {
+        outline: 3px solid color(primary, 200);
+    }
+}
+
+:host([variant='primary']) button,
+:host([selected][variant='primary']) button:not([disabled]) {
+    border-color: transparent;
+
+    &:hover {
+        background: color(primary, 400);
+        color: contrast-color(primary, 400);
+    }
+
+    &:focus {
+        background: color(primary, 500);
+        color: contrast-color(primary, 500);
+        outline: 3px solid color(primary, 200);
+    }
+}
+
+:host([variant='primary']) button[disabled] {
+    background: color(primary, 200);
+    color: #fff;
```

Comment:
Use the `rem` function throughout this file and don't hardcode the `color` values like in this instance. Try to land on a `contrast-color` value from the palette.